Source/player.cpp

```diff
@@ -789,7 +789,7 @@ bool PlrHitMonst(int pnum, int monsterId, bool adjacentDamage = false)
 {
 	int hper = 0;
 
-	if (monsterId < 0 || monsterId >= MaxMonsters) {
+	if (static_cast<size_t>(monsterId) >= MaxMonsters) {

```

Comment:
All calls to `PlrHitMonst()` verifies that `monsterId` is positive, so we can just make it size_t

Reply:
Done

Reply:
Didn't expect that you would fully remove the check, but on further inspection it looks safe to do that as well :sweat_smile: